hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/ha/TestObserverNode.java

```diff
@@ -583,7 +583,6 @@ public void testMkdirsRaceWithObserverRead() throws Exception {
       futures[i] = threadPool.submit(new MkDirRunner(conf2, clientStates[i]));
     }
 
-    Thread.sleep(150); // wait until mkdir is logged
```

Comment:
Can we just use GenricTestUtils.waitFor for this case, wait until mkdir gets actually gets logged?

Reply:
> Can we just use GenricTestUtils.waitFor for this case, wait until mkdir gets actually gets logged?
Thanks for your suggestion

Reply:
> Can we just use GenricTestUtils.waitFor for this case, wait until mkdir gets actually gets logged?
@ayushtkn Thanks for your suggestion. I have already adjusted it